modules/ROOT/pages/publish-apis.adoc

```diff
@@ -4,70 +4,114 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-API Community Manager publishes APIs from your organization's instance of Exchange so they can be accessed by your organization's developer community.
+An ACM community administrator imports APIs, new API versions, and minor API updates from their Exchange organization to their community, reviews them, and publishes them.
 
-== List your community's APIs
+== Import APIs
 
-Open the *Manage API Publication*. This tab displays lists of *APIs in this Community* and *APIs in Exchange*.
+Open the ACM administrator interface and select the *Manage APIs* tab at the top left.
 
-Each of your Anypoint organization's APIs are shown in one of the lists.
+On this tab you can manage what APIs from Exchange are presented in your community and their metadata and visibility.
```

Comment:
Maybe what APIs are available from Exchange . 

On this tab you can manage the APIs in your community, including their metadata, and visibility.

Do we need to say they are available from Exchange? If so, ignore my comment.

Reply:
I think it's best as "On this tab you can manage the APIs in your community, including their metadata and visibility." Fixed.